src/pages/guides/index.md

```diff
@@ -270,7 +271,7 @@ As mentioned earlier, I/O Events transforms your event payload and adds an addit
 Upon receiving a request, you must do the below for leveraging the enhanced security measures
 
 - verify you are the actual recipient of the event using the new `recipient_client_id` field available in the payload
-- once verified, your app should fetch the public key using the cloud front url present in the header
+- once verified, your app should fetch the public key by forming and validating the url using the adobe domain adobeioevents.com and the relative path received from the request header.
 - after downloading the public key set it in the cache with cache expiry of `not more than 24h` 
 
 You can also consider implementing a retry mechanism to call public key urls in case of any transient error that might occur.  
```

Comment:
If validation of any **one** of the signatures is successful, then the event can be considered as valid. This is to take care of transient key-rotation scenarios. Right?

Reply:
_nice to have_ - An additional block diagram of the validation steps will really help the consumers. Something in a pictorial form. wdyt?

Verify `recipient_client_id` -> Download and cache public keys -> Verify digital signature

Reply:
done

Reply:
Great!